pkg/services/object/get/assembly_v2.go

```diff
@@ -114,6 +114,7 @@ func (exec *execCtx) rangeFromLink(link objectSDK.Link) bool {
 
 			if i == first {
 				rngPerChild.SetOffset(uint64(firstOffset))
+				rngPerChild.SetLength(uint64(child.ObjectSize()) - uint64(firstOffset))
```

Comment:
I thought that unset length implies `[x:]` semantics.

Reply:
> I thought that unset length implies [x:] semantics.

me too. also, i could not imagine that it may be a bug but only a small session test will fail, not the range tests itself